test/trustedresources.go

```diff
@@ -34,13 +36,17 @@ import (
 	"github.com/sigstore/sigstore/pkg/cryptoutils"
 	"github.com/sigstore/sigstore/pkg/signature"
 	"github.com/tektoncd/pipeline/pkg/apis/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
-
+	"github.com/theupdateframework/go-tuf/encrypted"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fakek8s "k8s.io/client-go/kubernetes/fake"
 	"knative.dev/pkg/logging"
 )
 
+// TODO(#5527): refactor those into an internal pkg
```

Comment:
#5527 doesn't seem to say anything about an internal pkg, does it make sense to open a separate issue about it?

Reply:
yeah sure!